requirements_test.txt

```diff
@@ -2,6 +2,7 @@
 # make new things fail. Manually update these pins when pulling in a
 # new version
 asynctest==0.12.2
+attrs==18.2.0
```

Comment:
This shouldn't be necessary, it's already part of the core dependencies.

Reply:
The problem: when I do a `pip install -r requirements_test_all.txt` without this entry, `attrs` is downgraded to some lower version, probably because of some sub-module's requirements.